CHANGELOG.md

```diff
@@ -4,10 +4,21 @@ All notable changes to this project are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/airbnb/epoxy-ios/compare/0.9.0...HEAD)
+## [Unreleased](https://github.com/airbnb/epoxy-ios/compare/0.10.0...HEAD)
+
+### Changed
+- ...
+
+### Fixed
+- ...
+
+## [0.10.0](https://github.com/airbnb/epoxy-ios/compare/0.9.0...0.10.0) - 2023-06-29
```

Comment:
Should we release these changes as 0.10.0 after merging this PR or should I leave it under "Unreleased"? (Somebody else would have to cut the release since I don't have write access)

Reply:
@calda I'm wondering if we should cut a release before these changes, and then do a new release after these changes have been merged so we can include the other non-breaking changes currently unreleased?

Reply:
Good idea, that makes sense. We could release `0.9.1` and then `0.10.0`?

Reply:
Well, looking at the CHANGELOG we've only every cut releases like `0.6.0`, `0.7.0`, `0.8.0`, `0.9.0`. So I guess this should be 0.10.0 without the breaking changes and then 0.11.0 with the breaking changes?

Reply:
Here you go: https://github.com/airbnb/epoxy-ios/pull/144